charts/seed-bootstrap/templates/aggregate-prometheus/config.yaml

```diff
@@ -132,6 +132,28 @@ data:
       metric_relabel_configs:
 {{ include "prometheus.keep-metrics.metric-relabel-config" .Values.allowedMetrics.fluentbit| indent 6 }}
 
+    - job_name: fluent-bit-output-plugin
+      honor_labels: false
+      kubernetes_sd_configs:
+      - role: endpoints
+        namespaces:
+          names: [ garden ]
+      relabel_configs:
+      - target_label: __metrics_path__
+        replacement: /metrics
```

Comment:
https://prometheus.io/docs/prometheus/latest/configuration/configuration/#scrape_config
`/metrics` is the default metrics path, so you might be able to omit these 2 lines